cyences_app_for_splunk/default/macros.conf

```diff
@@ -1095,3 +1103,13 @@ iseval = 0
 [cs_cisco_ios_device_failed_login_filter]
 definition = search *
 iseval = 0
+
+
+# Cyences CIM indexes 
```

Comment:
Put this in the above section.

In macros.conf there are 3 sections in order:
- macros, we don't want user to update manually (this type of macros)
- internal filter macros
- filter macros and other macros like limits and stuff that user will update